app/forms/provider_interface/interview_wizard.rb

```diff
@@ -20,7 +20,8 @@ class InterviewWizard
                                        unless: ->(c) { %i[date time].any? { |d| c.errors.keys.include?(d) } }
     validate :date_after_rbd_date, if: %i[date_and_time date_and_time_in_future]
     validates :provider_id, presence: true, if: %i[application_choice provider_user multiple_application_providers?]
-    validates :location, presence: true
+    validates :location, presence: true, length: { maximum: 10240 }
```

Comment:
Although the maximum has been set, whilst testing locally this isn't mitigating the following error: 
```
2021-02-15 17:04:44 +0200 HTTP parse error, malformed request ("GET /provider/applications/17/interviews/new/check" - (-)): #<Puma::HttpParserError: HTTP element QUERY_STRING is longer than the (1024 * 10) allowed length (was 15164)>
```
As it isn't even reaching the validation stage yet, and falls over before that. I was wondering around us using POST vs GET which bypasses this issue? Let me know your thoughts

Reply:
Yes you're right, and the validation wouldn't even prevent that – it's possible to be under that number and still cause the error. The number here is chosen to match other free text fields in the API.

I'll card up and we can fix this bug separately